tests/integration/test_install_markers.py

```diff
@@ -167,8 +165,9 @@ def test_environment_variable_value_does_not_change_hash(pipenv_instance_private
             assert lock_hash is not None
             assert lock_hash == project.calculate_pipfile_hash()
 
-            # sanity check on pytest
-            assert 'PYPI_USERNAME' not in str(pipfile.load(p.pipfile_path))
+            # This line seems wrong. Why would pipenv install expand the variable and
```

Comment:
I agree -- lets just axe those assertions and not leave in the comments.

Reply:
Sure. Just wanted you to be aware of these things. Now that you noticed we can
remove it.